dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3555,6 +3557,69 @@ def setUpClass(cls):
                 cls.data, len(cls.data), options=profiler_options_full
             )
 
+    def test_add_profiles_null_count_not_enabled(self):
```

Comment:
based on the name I'd expect both `row_statistics.null_count.is_enabled` set to False...

Reply:
this should be: `test_adding_profiles_of_mismatched_null_count_options`

Reply:
you can test this error message below by creating a new options config in `test_add_profilers_row_statistics_options` [here](https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/tests/profilers/test_profile_builder.py#L3890)

Reply:
- `test_adding_profiles_of_mismatched_null_count_options` and  `test_add_profiles_null_count_not_enabled` can be removed since their functionality is being added to `test_add_profilers_row_statistics_options`. 

There needs to be tests to check the base conditions for `_get_row_is_null_ratio()` and `_get_row_has_null_ratio()`. 

- The test to check when `None` is [this](https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/tests/profilers/test_profile_builder.py#L3618). It already exists, it just needs to be updated so the options include `"row_statistics.null_count.is_enabled": False,`. The test to check when `0` is returned can be similar to [this](https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/tests/profilers/test_profile_builder.py#L4077). 
- There will need to be two test to check when `0` is returned, one for `_get_row_is_null_ratio()` and one for `_get_row_has_null_ratio()`. You can follow a similar format as [this](https://github.com/capitalone/DataProfiler/blob/main/dataprofiler/tests/profilers/test_profile_builder.py#L4077) test.